src/test/java/picard/analysis/CollectWgsMetricsTestUtils.java

```diff
@@ -51,7 +52,7 @@
  * Contains util methods for CollectWgsMetricsTest, CollectWgsMetricsWithNonZeroCoverageTest
  */
 
-public class CollectWgsMetricsTestUtils {
+public class CollectWgsMetricsTestUtils extends CommandLineProgramTest{
```

Comment:
```suggestion
public class CollectWgsMetricsTestUtils extends CommandLineProgramTest {
```

Reply:
Is this `extends` necessary? If all you need is `CHR_M_REFERENCE` you can reference that directly without inheriting.

Reply:
right.